agoradatatools/etl/load.py

```diff
@@ -114,18 +122,15 @@ def df_to_json(df: pd.core.frame.DataFrame, filename: str):
     try:
         df = df.replace({np.nan: None})
 
-        df_as_dict = df.to_dict(orient='records')
+        df_as_dict = df.to_dict(orient="records")
 
-        temp_json = open("./staging/" + filename, 'w+')
-        json.dump(df_as_dict, temp_json,
-                 cls=NumpyEncoder,
-                 indent=2)
+        temp_json = open("./staging/" + filename, "w+")
```

Comment:
Can we add a TODO: to use the `with open(...` context manager?

Reply:
added